Demo/Application/Features/ShopperInsightsViewController.swift

```diff
@@ -13,6 +13,8 @@ class ShopperInsightsViewController: PaymentButtonBaseViewController {
     lazy var payPalVaultButton = createButton(title: "PayPal Vault", action: #selector(payPalVaultButtonTapped))
     lazy var venmoButton = createButton(title: "Venmo", action: #selector(venmoButtonTapped))
     
+    private var shopperSessionID = "test-shopper-session-id"
```

Comment:
Can we also pass this in the shopper insights client in this class?
```suggestion
    private let shopperSessionID = "test-shopper-session-id"
```